### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by zakari-abdoul** [pdfminer/pdfminer.six#757]
Poorly documented, this is unfortunate.

**Comment by ldo** [comment]
I see some reasonably good docstrings in, for example, the `high_level` module. Worth copying and pasting some of that into the docs?

**Comment by KunalGehlot** [comment]
@ldo 
> I see some reasonably good docstrings in, for example, the `high_level` module. Worth copying and pasting some of that into the docs?

It should be a good start, but I think you should use that as a reference and create the definitions yourself. So that it is more understood by the users.

**Comment by pietermarsman** [comment]
Agreed! The docstrings should be used to document specific classes and methods. These could be included in the online docs as reference, but it is only part of the documentation. Other parts usually are tutorials (how to get started, how to solve common problems) and topics (background info).
